src/python/espressomd/interactions.pyx

```diff
@@ -1700,6 +1700,15 @@ cdef class BondedInteraction:
     def __reduce__(self):
         return (self.__class__, (self._bond_id,))
 
+    def __getstate__(self):
+        # for most bonds, we only need to pickle arguments passed to the
```

Comment:
this function should not be needed. please adapt the existing ```__reduce__``` function if necessary.